autogpt_platform/frontend/package.json

```diff
@@ -74,29 +79,40 @@
   "devDependencies": {
     "@chromatic-com/storybook": "^3.2.2",
     "@playwright/test": "^1.48.2",
-    "@storybook/addon-essentials": "^8.4.1",
-    "@storybook/addon-interactions": "^8.4.1",
-    "@storybook/addon-links": "^8.4.1",
-    "@storybook/addon-onboarding": "^8.4.1",
-    "@storybook/blocks": "^8.4.1",
-    "@storybook/nextjs": "^8.4.1",
+    "@storybook/addon-a11y": "^8.3.5",
+    "@storybook/addon-essentials": "^8.3.6",
+    "@storybook/addon-interactions": "^8.3.6",
+    "@storybook/addon-links": "^8.3.6",
+    "@storybook/addon-onboarding": "^8.3.6",
+    "@storybook/blocks": "^8.3.6",
+    "@storybook/nextjs": "^8.3.6",
```

Comment:
all kinds of crap got bumped by dependabot for security and various other reasons

Reply:
Just merged the version updates in